source/extensions/filters/network/wasm/BUILD

```diff
@@ -0,0 +1,42 @@
+load(
+    "//bazel:envoy_build_system.bzl",
+    "envoy_cc_extension",
+    "envoy_cc_library",
+    "envoy_extension_package",
+)
+
+licenses(["notice"])  # Apache 2
+
+envoy_extension_package()
+
+# Public docs: docs/root/configuration/network_filters/wasm_filter.rst
+
+envoy_cc_library(
+    name = "wasm_filter_lib",
+    srcs = ["wasm_filter.cc"],
+    hdrs = ["wasm_filter.h"],
+    deps = [
+        "//include/envoy/server:filter_config_interface",
+        "//include/envoy/upstream:cluster_manager_interface",
+        "//source/extensions/common/wasm:wasm_lib",
+        "//source/extensions/filters/network:well_known_names",
+        "@envoy_api//envoy/extensions/filters/network/wasm/v3:pkg_cc_proto",
+    ],
+)
+
+envoy_cc_extension(
+    name = "config",
+    srcs = ["config.cc"],
+    hdrs = ["config.h"],
+    security_posture = "unknown",
```

Comment:
also alpha

Reply:
What is the syntax?  "unknown alpha" "unknown, alpha" or just "alpha" ?

Reply:
See https://github.com/envoyproxy/envoy/blob/da403fa7b67b9331ca1d129a4035950d2e23c752/source/extensions/grpc_credentials/file_based_metadata/BUILD#L19.

Reply:
Done